src/main/java/com/ziro/onepassword/sdk/OnePasswordSecrets.java

```diff
@@ -0,0 +1,118 @@
+package com.ziro.onepassword.sdk;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.ziro.espresso.fluent.exceptions.SystemUnhandledException;
+import com.ziro.espresso.javax.annotation.extensions.NonNullByDefault;
+import com.ziro.espresso.okhttp3.OkHttpClientFactory;
+import com.ziro.espresso.okhttp3.SynchronousCallAdapterFactory;
+import com.ziro.espresso.streams.MoreCollectors;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.X509TrustManager;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.Value;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+@Value
+@NonNullByDefault
+public class OnePasswordSecrets {
```

Comment:
Rename this if you want to `OnePasswordConnector`

Reply:
Done